packages/features/embed/Embed.tsx

```diff
@@ -516,7 +516,6 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
   const iframeRef = useRef<HTMLIFrameElement>(null);
   const dialogContentRef = useRef<HTMLDivElement>(null);
   const flags = useFlagMap();
```

Comment:
Are other flags being used in this file? Otherwise we can remove useFlags as well.